db/migrate/20200404095742_movie_likes.rb

```diff
@@ -0,0 +1,13 @@
+class MovieLikes < ActiveRecord::Migration[6.0]
+  def change
+    create_table :movie_likes do |t|
+
+      #movies foreign_key
+      t.references :movie, foreign_key: true
+
+      #users foreign_key
+      t.references :user, foreign_key: true
+      
```

Comment:
Rubocop: Add timestamps when creating a new table
ie add `t.timestamps` after the t.references lines